mizar/obj/net.py

```diff
@@ -120,6 +126,12 @@ def set_status(self, status):
     def get_gw_ip(self):
         return str(self.cidr.get_ip(1))
 
+    def set_remote_deployed(self, remote_deployed):
+        self.remote_deployed = remote_deployed
+
+    def set_cluster_gateway(self, cluster_gateway):
```

Comment:
param default empty?

Reply:
Added

Reply:
Does not look like it has been added.

Reply:
It is added. The comment is Outdated. Please refresh the PR and you will see changes.

Reply:
I refreshed it. Still don't see it.

Reply:
As to virutal, it is added. As to cluster_gateway, its default is empty.